slider.js

```diff
@@ -2,12 +2,12 @@
 	'use strict';
 	var min = $(".slider__range").attr("min");
 	var max = $(".slider__range").attr("max");
-	var horizontal;
+	var vertical;
 
 	window.updateOutput = function (figure) {
 		$(".slider__banana").html(figure);
-		horizontal = figure/max*($(".eating__slider").width()-$(".slider__reel").width()) + 'px';
-		$(".slider__banana, .slider__reel").css({left: horizontal});
+		vertical = figure/max*($(".eating__slider").height()-$(".slider__reel").height()) + 'px';
+		$(".slider__banana, .slider__reel").css({bottom: vertical});
```

Comment:
indentation.. Is this code copy-pasted? if from stack overflow should leave a link..